bin/renormalize_enumchrons.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require "dotenv"
+Dotenv.load(".env")
+
+$LOAD_PATH.unshift(File.join(File.dirname(__FILE__), "..", "lib"))
+require "bundler/setup"
+require "utils/waypoint"
+require "utils/ppnum"
+require "cluster"
+require "services"
+
+Mongoid.load!("mongoid.yml", ENV["MONGOID_ENV"])
+
+def renormalize(item)
+  item.normalize_enum_chron
+  item.save
+end
+
+def records_with_enum_chrons
+  return enum_for(:records_with_enum_chrons) unless block_given?
+
+  Cluster.where("$or": [{ "holdings.enum_chron": { "$ne": "" } },
```

Comment:
I did some experiments and I'm pretty sure this will match clusters that have any of the following:
* fulfill the criteria you want
* have ht_items or ht_holdings unset (e.g., one ht_item with no enum_chron, and no holdings at all)
* have ht_items or ht_holdings set to the empty array (because something got moved?)

Not a huge deal in this case because they'll be no-opped in the loop, but useful to know in other contexts, I would think.